src/app/Instance/InstanceDetails/InstanceDetails.tsx

```diff
@@ -63,45 +65,72 @@ export const InstanceDetails = ({
         </DrawerActions>
       </DrawerHead>
       <DrawerContentBody>
-        <DescriptionList isHorizontal isCompact>
-          <DescriptionListGroup>
-            <DescriptionListTerm>{t("common.id")}</DescriptionListTerm>
-            <DescriptionListDescription data-ouia-component-id="instance-details-id">
-              {instance.id}
-            </DescriptionListDescription>
-          </DescriptionListGroup>
-          <DescriptionListGroup>
-            <DescriptionListTerm>{t("common.owner")}</DescriptionListTerm>
-            <DescriptionListDescription data-ouia-component-id="instance-details-owner">
-              {instance.owner}
-            </DescriptionListDescription>
-          </DescriptionListGroup>
-          <DescriptionListGroup>
-            <DescriptionListTerm>{t("common.url")}</DescriptionListTerm>
-            <DescriptionListDescription>
-              <ClipboardCopy
-                data-ouia-component-id="instance-details-endpoint"
-                isReadOnly
-                hoverTip={t("common.copy")}
-                clickTip={t("common.copied")}
-              >
-                {instance.endpoint}
-              </ClipboardCopy>
-            </DescriptionListDescription>
-          </DescriptionListGroup>
-          <DescriptionListGroup>
-            <DescriptionListTerm>{t("common.submittedAt")}</DescriptionListTerm>
-            <DescriptionListDescription data-ouia-component-id="instance-details-submitted-date">
-              {instance.submitted_at && formatDate(instance.submitted_at)}
-            </DescriptionListDescription>
-          </DescriptionListGroup>
-          <DescriptionListGroup>
-            <DescriptionListTerm>{t("common.publishedAt")}</DescriptionListTerm>
-            <DescriptionListDescription data-ouia-component-id="instance-details-published-date">
-              {instance.published_at && formatDate(instance.published_at)}
-            </DescriptionListDescription>
-          </DescriptionListGroup>
-        </DescriptionList>
+        <Stack hasGutter>
```

Comment:
Trying to put a bit more space between the two sections, I think we could have two `DrawerContentBody`'s instead of a wrapping `Stack`. 

Actually looking at the [docs](https://www.patternfly.org/v4/components/drawer#stacked-content-body-elements) I've learned that inside the drawer panel it should be a `DrawerPanelBody`. 
So there could be two `DrawerPanelBody`'s, one with the basic properties and one with the ingress stuff.

Reply:
Done: https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/108/commits/4d56538b186896174c66886e030fa66e6ba9a738